CHANGELOG.md

```diff
@@ -1,5 +1,28 @@
 # Beeline Changelog
 
+## [2.0.0] - 2022-03-21
+
+### !!! Breaking Changes !!!
+
+The beeline will no longer propagate `dataset` configuration across distributed traces. To ensure distributed traces continue to send data to the appropriate dataset either:
+1. Configure a `dataset` in every instrumented application (e.g. using the `honeycomb.beeline.dataset` property). This is likely already the case for most users.
```

Comment:
nit: Unsure of wording for those who don't understand what's changing. Maybe instead of "This is likely already the case for most users.", try something like "This is likely already set this way for most users."?

Reply:
updated 🌷